openmetadata-ui/src/main/resources/ui/src/components/Ingestion/Ingestion.component.tsx

```diff
@@ -95,6 +102,24 @@ const Ingestion: React.FC<IngestionProps> = ({
     setSearchText(searchValue);
   };
 
+  const [list, setList] = useState<Array<IngestionPipeline>>([]);
```

Comment:
@Ashish8689 , Are we not getting ingestionList in the props?

Reply:
@Sachin-chaurasiya we, are updating it.